src/lib.rs

```diff
@@ -12,9 +12,7 @@ mod types;
 /// The cargo wrapper so that `cargo check-all-features ...` will work, since it internally invokes `check-all-features` with itself
 /// as the first argument
 enum CargoCli {
-    #[command(name = "check-all-features")]
-    #[command(alias = "build-all-features")]
-    #[command(alias = "test-all-features")]
+    #[command(name = "all-features")]
```

Comment:
issue: again, backwards compatability. You may need multiple options types, which share internals. Unfortunate, but whatever.